qsiprep/interfaces/dwi_merge.py

```diff
@@ -151,6 +163,16 @@ def _run_interface(self, runtime):
         return runtime
 
 
+def combine_metadata(scan_list, metadata_dict):
+    """Create a merged metadata dictionary.
+
+    Most importantly, combine the slice timings in some way. Here the slice
+    metadata is taken from the first scan.
+
+    """
+    return metadata_dict[scan_list[0]]
```

Comment:
What about raising a warning if `SliceTiming` doesn't match across scans? That way, at least users will be aware of an issue.

Reply:
That's still in my TODOs for this PR. I was going to have that somewhere in utils/grouping.py so there is an early error